README.md

```diff
@@ -80,24 +126,29 @@ oc login <openshift url>
 
 Utilize the following steps to deploy the project
 
-1. Clone the repository
+1. Clone the repository and checkout v2 branch
 
     ```
-    git clone https://github.com/matallen/community-ninja-board
+    git clone https://github.com/redhat-cop/ninja-board
+    git checkout v2
     ```
 
 2. Change into the project directory and utilize Ansible Galaxy to retrieve required dependencies
 
     ```
     ansible-galaxy install -r requirements.yml --roles-path=galaxy
-    ``` 
- 
-3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
+    ```
+
+3. Ensure the variables have been updated in the _.applier/inventory/group_vars/all.yml_ ansible params file to the desired values
 
 4. Execute the _openshift-applier_
 
     ```
-    ansible-playbook -i .applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment"
+    ansible-playbook -i .applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment" -e include_tags=v2
     ```
 
-Once complete, all of the resources should be available in OpenShift
+    ***NOTES***
+    * If you have not yet created the openshift dev project, you will need to change the `include_tags` argument to `include_tags=project-req-dev,v2`
+    * If you are not behind RedHat Network and would like access to services that use ldap, you will need to change the `include_tags` argument to `include_tags=ldap-rbac,ldap,v2`. This will ***FAIL*** if you do not have sufficient priveleges. If you don't have access to perform this, ask a cluster administrator.
+
+Once complete, all of the v2 resources should be available in OpenShift
```

Comment:
There is a piece missing here regarding the deployment of the included [Jenkins template](https://github.com/redhat-cop/ninja-board/blob/c5fbc0c36efc59e848888caa803552afae568a8f/.openshift/templates/jenkins-ephemeral.yml). I would also recommend removing lines 25,27,28,29,30 as they contain runtime data that should never be included.

Deployment of jenkins requires that `cicd` be added to the list of `include_tags`

Reply:
Done